.gitignore

```diff
@@ -2,3 +2,5 @@
 /dist/**/*
 .env
 .DS_Store
+.DS_store
```

Comment:
does this file come back as both `.DS_Store` and `.DS_store`? if not, please remove the one that's irrelevant